app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/MySessionsFragment.java

```diff
@@ -0,0 +1,133 @@
+package io.github.droidkaigi.confsched2017.view.fragment;
+
+import android.content.Context;
+import android.databinding.ObservableList;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import javax.inject.Inject;
+
+import io.github.droidkaigi.confsched2017.R;
+import io.github.droidkaigi.confsched2017.databinding.FragmentMySessionsBinding;
+import io.github.droidkaigi.confsched2017.databinding.ViewMySessionBinding;
+import io.github.droidkaigi.confsched2017.model.MySession;
+import io.github.droidkaigi.confsched2017.view.activity.SessionDetailActivity;
+import io.github.droidkaigi.confsched2017.view.customview.BindingHolder;
+import io.github.droidkaigi.confsched2017.view.customview.ObservableListRecyclerAdapter;
+import io.github.droidkaigi.confsched2017.view.customview.itemdecoration.DividerItemDecoration;
+import io.github.droidkaigi.confsched2017.viewmodel.MySessionViewModel;
+import io.github.droidkaigi.confsched2017.viewmodel.MySessionsViewModel;
+import io.reactivex.disposables.CompositeDisposable;
+
+
+public class MySessionsFragment extends BaseFragment {
+
+    public static final String TAG = MySessionsFragment.class.getSimpleName();
+
+    @Inject
+    MySessionsViewModel viewModel;
+
+    @Inject
+    CompositeDisposable compositeDisposable;
```

Comment:
Ah, sorry. It's my misunderstanding 🙇 
Maybe this `compositeDisposable` is unnecessary here.